kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/installation/vault/Vault.java

```
@@ -0,0 +1,147 @@
+
+        return exitCode == 0 &&
+                output.toString().toLowerCase().contains("vault is initialized");
+    }
+
+    /**
+     * Deploy.
+     *
+     */
+    public void deploy() {
+        LOGGER.info("Deploy HashiCorp Vault in {} namespace", deploymentNamespace);
+        if (isDeployed()) {
+            LOGGER.warn("Skipping Vault deployment. It is already deployed!");
+            return;
+        }
+
+        Map<String, String> values = new HashMap<>();
+        // server
+        values.put("server.dev.enabled", "true");
+        values.put("server.dev.devRootToken", VAULT_ROOT_TOKEN);
+        values.put("server.ha.enabled", "false");
+        values.put("server.updateStrategyType", "RollingUpdate");
+        values.put("server.service.type", "NodePort");
+        // injector
+        values.put("injector.enabled", "false");
+
+        ResourceManager.helmClient().addRepository(VAULT_HELM_REPOSITORY_NAME, VAULT_HELM_REPOSITORY_URL);
+        ResourceManager.helmClient().namespace(deploymentNamespace).install(VAULT_HELM_CHART_NAME, VAULT_SERVICE_NAME, "latest", values);
+
+        DeploymentUtils.waitForDeploymentRunning(deploymentNamespace, VAULT_POD_NAME, Duration.ofMinutes(1));
+
+        configureVault(deploymentNamespace);
+    }
+
+    private void configureVault(String deploymentNamespace) {
+        LOGGER.info("Enabling transit in vault instance");
+        String loginCommand = VAULT_CMD + " login " + VAULT_ROOT_TOKEN;
+        String transitCommand = VAULT_CMD + " secrets enable transit";
+
+        int exitCode = kubeClient().getClient().pods()
+                .inNamespace(deploymentNamespace)
+                .withName(VAULT_POD_NAME)
+                .exec("sh", "-c", String.format("%s && %s", loginCommand, transitCommand)).exitCode().join();
+
+        if (exitCode != 0) {
+            String errorLog = kubeClient().logsInSpecificNamespace(deploymentNamespace, VAULT_POD_NAME);
```

The intent of my previous comment here was that you should log from stderr/stdout from the failed exec command, rather than the container log.